From 9510d8470ec9397bab4805c24ddf356431b6b327 Mon Sep 17 00:00:00 2001
From: hehmde <hehmde@ukr.net>
Date: Wed, 30 Sep 2020 01:46:18 +0300
Subject: [PATCH 1/2] fix truncation error

---
 calculator.cpp | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/calculator.cpp b/calculator.cpp
index b91afea..f04ea27 100644
--- a/calculator.cpp
+++ b/calculator.cpp
@@ -2,7 +2,8 @@
 
 int Calculator::Add (double a, double b)
 {
-    return a + b;
+	return a + b + 0.5;
+
 }
 
 int Calculator::Sub (double a, double b)
-- 
2.28.0.windows.1

