From c2aef762f676b650b82999b67ab590115ead6c48 Mon Sep 17 00:00:00 2001
From: hehmde <hehmde@ukr.net>
Date: Wed, 30 Sep 2020 01:47:20 +0300
Subject: [PATCH 2/2] formatting: use tabs instead of spaces

---
 calculator.cpp | 3 +--
 calculator.h   | 6 +++---
 2 files changed, 4 insertions(+), 5 deletions(-)

diff --git a/calculator.cpp b/calculator.cpp
index f04ea27..d10f529 100644
--- a/calculator.cpp
+++ b/calculator.cpp
@@ -3,10 +3,9 @@
 int Calculator::Add (double a, double b)
 {
 	return a + b + 0.5;
-
 }
 
 int Calculator::Sub (double a, double b)
 {
-    return Add (a, -b);
+	return Add (a, -b);
 }
diff --git a/calculator.h b/calculator.h
index 3740907..d59d596 100644
--- a/calculator.h
+++ b/calculator.h
@@ -3,9 +3,9 @@
 
 class Calculator
 {
-    public:
-        int Add (double, double);
-        int Sub (double, double);
+	public:
+		int Add (double, double);
+		int Sub (double, double);
 };
 
 #endif//CALCULATOR_H
-- 
2.28.0.windows.1

